gfx_scene.h

```diff
@@ -379,7 +377,8 @@ bool gfxSceneSetMeshMetadata(GfxScene scene, uint64_t mesh_handle, GfxMetadata c
 
 struct GfxInstance
 {
-    GfxConstRef<GfxMesh> mesh;
+    GfxConstRef<GfxMesh>     mesh;
+    GfxConstRef<GfxMaterial> material;
```

Comment:
Material is now in GfxInstance instead of GfxMesh as this allows for same mesh to have multiple materials in different instances